core/src/main/scala/za/co/absa/spline/core/harvester/operationBuilders.scala

```diff
@@ -166,6 +166,10 @@ class SaveJDBCCommandNodeBuilder
     writeMetrics = writeMetrics,
     readMetrics = readMetrics
   )
+
+  override def ignoreLineageWrite:Boolean = {
+    false
```

Comment:
Can't we detect `SaveMode.Ignore` in case of JDBC write? Or it is not supported?

Reply:
This is not detecting SaveMode.Ignore, this should detect if any data were written.